### ISSUE TITLE
Simple docker pull command fails because repo does not have latest image tagged

### ISSUE THREAD

**Issue by aholland** [instructure/canvas-lms#1330]
### Summary:
Pull command fails despite being explicitly stated at https://hub.docker.com/r/instructure/canvas-lms/ 

### Steps to reproduce:
 1. Go to https://hub.docker.com/r/instructure/canvas-lms/
 2. Copy the docker pull command explicitly stated on that page ("docker pull instructure/canvas-lms)
 3. Execute that command

### Expected behavior:
latest image is downloaded by docker

### Actual behavior:
Error response from daemon: manifest for instructure/canvas-lms:latest not found

### Additional notes:
This is particularly unfriendly to novices. That's bad PR. All you need to do is label the latest image with "latest".

**Comment by themitigater** [comment]
Use docker pull instructure/canvas-lms:(master/stable/beta).

**Comment by aholland** [comment]
> Use docker pull instructure/canvas-lms:(master/stable/beta).
Sure that works - but it's not ideal to put a command on Docker Hub page (https://hub.docker.com/r/instructure/canvas-lms/) complete with a copy-to-clipboard button, and then have that command not work in that exact form.

**Comment by themitigater** [comment]
Hey Holland, 
I wasn't suggesting that the comment be added to the docker hub page, but instead edit the comment when trying to actually pull the image.

**Comment by aholland** [comment]
Yes and that's very kind of you, but it doesn't address the underlying issue, which is that the instructions taken directly from the GitHub page do not work as presented.

**Comment by ccutrer** [comment]
Please use [the mailing list](https://groups.google.com/g/canvas-lms-users) for installation and operational issues.
